### PR TITLE
Update deprecated test aliases

### PR DESCRIPTION
**Pull request by ngash**
- change ``failIf`` to ``assertFalse``
- change ``asertEquals`` to ``assertEqual``
- change ``assert_`` to ``assertTrue``

https://docs.python.org/2/library/unittest.html#deprecated-aliases

### CODE REVIEW

File: tests/test_http_cookies.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def setUp(self):
         self.wrapped = WrappedResponse(self.response)
 
     def test_info(self):
-        self.assert_(self.wrapped.info() is self.wrapped)
+        self.assertTrue(self.wrapped.info() is self.wrapped)
```
Comment (kmike) [inline tests/test_http_cookies.py]: would you mind changing it to `assertIs`, as we're on it?

File: tests/test_http_request.py [hunk 2]
```diff
@@ -64,9 +64,9 @@ def test_headers(self):
         h = Headers({'key1': u'val1', u'key2': 'val2'})
         h[u'newkey'] = u'newval'
         for k, v in h.iteritems():
-            self.assert_(isinstance(k, bytes))
+            self.assertTrue(isinstance(k, bytes))
             for s in v:
-                self.assert_(isinstance(s, bytes))
+                self.assertTrue(isinstance(s, bytes))
```
Comment (kmike) [inline tests/test_http_request.py]: it'd be awesome to change it to assertIsInstance, as we're updating this code

File: tests/test_loader.py [hunk 3]
```diff
@@ -437,7 +437,7 @@ def test_join(self):
         self.assertRaises(TypeError, proc, [None, '', 'hello', 'world'])
         self.assertEqual(proc(['', 'hello', 'world']), u' hello world')
         self.assertEqual(proc(['hello', 'world']), u'hello world')
-        self.assert_(isinstance(proc(['hello', 'world']), six.text_type))
+        self.assertTrue(isinstance(proc(['hello', 'world']), six.text_type))
```
Comment (kmike) [inline tests/test_loader.py]: assertIsInstance

File: tests/test_loader.py [hunk 4]
```diff
@@ -482,29 +482,29 @@ def test_constructor_errors(self):
     def test_constructor_with_selector(self):
         sel = Selector(text=u"<html><body><div>marta</div></body></html>")
         l = TestItemLoader(selector=sel)
-        self.assert_(l.selector is sel)
+        self.assertTrue(l.selector is sel)
```
Comment (kmike) [inline tests/test_loader.py]: assertIs

File: tests/test_loader.py [hunk 5]
```diff
@@ -482,29 +482,29 @@ def test_constructor_errors(self):
...
 
         l.add_xpath('name', '//div/text()')
         self.assertEqual(l.get_output_value('name'), [u'Marta'])
 
     def test_constructor_with_selector_css(self):
         sel = Selector(text=u"<html><body><div>marta</div></body></html>")
         l = TestItemLoader(selector=sel)
-        self.assert_(l.selector is sel)
+        self.assertTrue(l.selector is sel)
```
Comment (kmike) [inline tests/test_loader.py]: assertIs

File: tests/test_utils_python.py [hunk 6]
```diff
@@ -156,27 +156,27 @@ def test_stringify_dict(self):
         d = {'a': 123, u'b': b'c', u'd': u'e', object(): u'e'}
         d2 = stringify_dict(d, keys_only=False)
         self.assertEqual(d, d2)
-        self.failIf(d is d2)  # shouldn't modify in place
-        self.failIf(any(isinstance(x, six.text_type) for x in d2.keys()))
-        self.failIf(any(isinstance(x, six.text_type) for x in d2.values()))
+        self.assertFalse(d is d2)  # shouldn't modify in place
```
Comment (kmike) [inline tests/test_utils_python.py]: assertIsNot

File: tests/test_utils_python.py [hunk 7]
```diff
@@ -156,27 +156,27 @@ def test_stringify_dict(self):
...
     @unittest.skipUnless(six.PY2, "deprecated function")
     def test_stringify_dict_tuples(self):
         tuples = [('a', 123), (u'b', 'c'), (u'd', u'e'), (object(), u'e')]
         d = dict(tuples)
         d2 = stringify_dict(tuples, keys_only=False)
         self.assertEqual(d, d2)
-        self.failIf(d is d2)  # shouldn't modify in place
-        self.failIf(any(isinstance(x, six.text_type) for x in d2.keys()), d2.keys())
-        self.failIf(any(isinstance(x, six.text_type) for x in d2.values()))
+        self.assertFalse(d is d2)  # shouldn't modify in place
```
Comment (kmike) [inline tests/test_utils_python.py]: assertIsNot

File: tests/test_utils_python.py [hunk 8]
```diff
@@ -156,27 +156,27 @@ def test_stringify_dict(self):
...
+        self.assertFalse(any(isinstance(x, six.text_type) for x in d2.values()))
 
     @unittest.skipUnless(six.PY2, "deprecated function")
     def test_stringify_dict_keys_only(self):
         d = {'a': 123, u'b': 'c', u'd': u'e', object(): u'e'}
         d2 = stringify_dict(d)
         self.assertEqual(d, d2)
-        self.failIf(d is d2)  # shouldn't modify in place
-        self.failIf(any(isinstance(x, six.text_type) for x in d2.keys()))
+        self.assertFalse(d is d2)  # shouldn't modify in place
```
Comment (kmike) [inline tests/test_utils_python.py]: assertIsNot

File: tests/test_utils_signal.py [hunk 9]
```diff
@@ -29,7 +29,7 @@ def test_send_catch_log(self):
         self.assertIn('error_handler', record.getMessage())
         self.assertEqual(record.levelname, 'ERROR')
         self.assertEqual(result[0][0], self.error_handler)
-        self.assert_(isinstance(result[0][1], Failure))
+        self.assertTrue(isinstance(result[0][1], Failure))
```
Comment (kmike) [inline tests/test_utils_signal.py]: assertIsInstance

File: tests/test_webclient.py [hunk 10]
```diff
@@ -326,7 +326,7 @@ def testNotFound(self):
         return getPage(self.getURL('notsuchfile')).addCallback(self._cbNoSuchFile)
 
     def _cbNoSuchFile(self, pageData):
-        self.assert_(b'404 - No Such Resource' in pageData)
+        self.assertTrue(b'404 - No Such Resource' in pageData)
```
Comment (kmike) [inline tests/test_webclient.py]: assertIn

File: tests/test_http_request.py [hunk 11]
```diff
@@ -64,9 +64,9 @@ def test_headers(self):
         h = Headers({'key1': u'val1', u'key2': 'val2'})
         h[u'newkey'] = u'newval'
         for k, v in h.iteritems():
-            self.assert_(isinstance(k, bytes))
+            self.assertTrue(isinstance(k, bytes))
```
Comment (kmike) [inline tests/test_http_request.py]: assertIsInstance

File: tests/test_http_cookies.py (hunk 1 above)
Comment (ngash) [inline tests/test_http_cookies.py]: Ok

**Comment by redapple** [comment]
Thank you @ngash !

**Pull request merged**